package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@netzstrategen/twig-asset",
+  "version": "1.0.0",
+  "description": "",
+  "main": "js/functions.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/netzstrategen/cue-twig-extensions.git"
+  },
+  "author": "Fabian Marz <fabian@netzstrategen.com>",
+  "license": "MIT",
```

Comment:
The license should match the one in composer.json

Reply:
Sorry, missed this 🙈